buildkite/scripts/replayer-test.sh

```diff
@@ -36,4 +36,6 @@ cd /workdir
 
 echo "Running replayer"
 mina-replayer --archive-uri postgres://postgres:$PGPASSWORD@localhost:5432/archive \
-	      --input-file $TEST_DIR/input.json --output-file /dev/null
+	      --input-file $TEST_DIR/input.json --output-file /dev/null \
+          --continue-on-error # DO NOT MERGE
```

Comment:
Says it should not be merged.

Reply:
It's reverted; I wanted to keep the commits around because there's useful information in the commit messages, and it's nice to re-cherry-pick each time.